src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
```

Comment:
<img width="1015" alt="Screenshot 2023-01-06 at 8 12 35 PM" src="https://user-images.githubusercontent.com/60295799/211073026-63298a6b-0a42-4d6f-8fd9-95b171206f5a.png">


check this [instruction](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)